### ISSUE TITLE
Idea: support for pre-rendered/cached content

### ISSUE THREAD

**Issue by zeroasterisk** [iron-meteor/iron-router#251]
As discussed [here](https://trello.com/c/69Jndpqf/47-page-load-time-optimization) it would be nice to have the ability to cache a rendered version of certain routes, such that they could be handed to the client "complete" while waiting on the data/assets/etc to load onto the client.

While this wouldn't fix everything, it would be a neat feature.

This is almost like a "loading" page, but it could be something more automatic than a custom loading page for every route...

If you are talking about a dynamic app, "loading" is about the best you could hope for... but for a basic blog type app, the content will likely me almost always rendered the same... if we could leverage that, it sounds like a win.

_(this is a brainstorm/idea at this point -- if "out of scope" or poorly conceived, just close this idea)_

**Comment by reustle** [comment]
Just wanted to add a reference to the `fast-render` package for those who end up here via google

https://atmospherejs.com/meteorhacks/fast-render
